dev/com.ibm.ws.cdi.interfaces/src/com/ibm/wsspi/cdi/extension/WebSphereCDIExtensionMetaData.java

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2015 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.cdi.spi;
+
+import java.util.Collections;
+import java.util.Set;
+
+import javax.enterprise.inject.spi.Extension;
+
+/**
+ * This is a *marker* interface for CDI Runtime extensions. Liberty features that wish to extend CDI will need to
+ * register a service under this interface.
+ *
```

Comment:
You need to use `<p>` instead of a blank line to get a new paragraph when writing javadoc that's going to be rendered as HTML (which this will because it's SPI).